contributors/devel/security-release-process.md

```diff
@@ -82,14 +112,47 @@ The communication to users should be actionable. They should know when to block
 
 **Fix Release Day** (Completed within 1-21 days of Disclosure)
 
-- The Release Managers will ensure all the binaries are built, publicly available, and functional before the Release Date.
-  - TODO: this will require a private security build process.
-- The Release Managers will create a new patch release branch from the latest patch release tag + the fix from the security branch. As a practical example if v1.5.3 is the latest patch release in kubernetes.git a new branch will be created called v1.5.4 which includes only patches required to fix the issue.
-- The Fix Lead will cherry-pick the patches onto the master branch and all relevant release branches. The Fix Team will LGTM and merge.
-- The Release Managers will merge these PRs as quickly as possible. Changes shouldn't be made to the commits even for a typo in the CHANGELOG as this will change the git sha of the already built and commits leading to confusion and potentially conflicts as the fix is cherry-picked around branches.
+- The Release Managers will ensure all the binaries are built,
+  publicly available, and functional before the Release Date.
+    - Note: we ship source with our binary releases, so as soon as binary
+      artifacts are publicly available, the fixes (and vulnerabilities) are
+      effectively public.
+  - CI testing of the private patch release branch should provide confidence in
+    the release.
+- The Release Manager will use release tooling (i.e. `anago`) to
+  - Compute the version for the new security patch release; in the previous
+    example, the `release-2.4.7` branch would produce a `v2.4.8` release.
+  - Verify that no public patch release has occurred since the security branch
+    was cut (in this example, make sure that `v2.4.8` has not been tagged
+    publicly).
+  - Perform a normal release build from the private release branch, staging
+    binary artifacts locally (or possibly in a private bucket).
+  - Push the new patch release tag to the private repo.
+  - The tooling should **not** upload any artifacts to public storage buckets or
+    create a draft release on the public GitHub repo.
+- As soon as we are ready to make the fix known publicly (TODO: when?),
+  the Release Manager will use tooling to
+  - Publish results.
+    - Upload binary artifacts to GCS, gcr.io, and GitHub. (TODO: are these
+      prebuilt from the previous step?)
+    - Update the `CHANGELOG.md` in the **public** `master` branch
+    - Create a new release on the **public** kubernetes GitHub
+    - Send email announcing the release
+  - Merge the private patch release branch into the **public** release branch
```

Comment:
We need to do both of these things:
* push the released tag to the public repo (so the released sha is available via the given tag)
* merge the private patch release branch into the public release branch as described